front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -55,6 +56,18 @@ const SimulationReportSheet = ({
     },
     [mapImageUrl]
   );
+  const getArrivalTime = (
```

Comment:
You can add an empty line before getArrivalTime :) 
```suggestion
  
  const getArrivalTime = (
```

Reply:
fixed